boot/bootutil/src/loader.c

```diff
@@ -81,8 +81,8 @@ static struct boot_loader_state boot_data;
 #define TARGET_STATIC
 #endif
 
-#if BOOT_MAX_ALIGN > 1024
```

Comment:
Previous values for undefined MCUBOOT_BOOT_COPY_REGION_SIZE should be kept

Reply:
I'm not sure that I follow. How are they being discarded?

In all prior versions `BOOT_MAX_ALIGN > 1024` is _never true_, so this condition can be removed entirely. And `BOOT_COPY_REGION_SIZE` is only defined iff `MCUBOOT_BOOT_COPY_REGION_SIZE` is defined, so by default,  if a user does not define `MCUBOOT_BOOT_COPY_REGION_SIZE` then 1024 will be used

Reply:
OK, Can you investigate CI failure?

Reply:
Sure thing, I will take a look later today